adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,75 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_4_1']
+=== Release 1.4.1, July 2019
+
+[id='sec.1_4_1.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.17.1.
+* Added option to deploy UAA from the SCF Helm chart. In new CAP deployments that do not require a separate UAA deployment, setting `enable.uaa=true` in the Helm values automatically creates a UAA StatefulSet within the SCF deployment. This removes the manual step of copying the UAA_CA_CERT during setup. In this configuration UAA uses the same database as SCF, reducing the memory footprint of the cluster. This option should be left as `false` (default) for upgrades of existing CAP clusters.
```

Comment:
* The second sentence is a mouthful and I struggle to keep everything in my mind until the end. So, moving the goal first and then adding the instruction at the end helps a little here: "If you are newly deploying CAP and do not require a separate UAA deployment, you can now generate a UAA StatefulSet within the SCF deployment. To do so, set `enable.uaa=true` in the Helm values. This means there is no need to copy the UAA_CA_CERT during setup." -- still not great, though.
* What is the exact definition of `separate deployment`? Should that be `deployment on separate node`?
* In the penultimate sentence, add a comma: `In this configuration, UAA uses...`

Reply:
To address the 2nd point, previously with CAP, UAA & SCF were deployable into separate namespaces by default. With 1.4.1, you can now deploy both components into the same namespace, which makes for an easier configuration for security setup.

I've implemented your suggestions and we can refine this section later.